packages/react-storage/src/components/StorageBrowser/displayText/en.ts

```diff
@@ -2,7 +2,12 @@
  * Temporarily just export strings until displayText strategy is finalized
  */
 export const displayText = {
+  actionDestinationPickerCurrentFolderSelected: 'Current folder selected:',
+  actionDestinationPickerNoMoreFolders: 'There are no folders under this path.',
+  actionDestinationPickerDefaultError: 'There was an error loading folders.',
```

Comment:
This level of verbosity is going to be extremely difficult for consumer to grok

Reply:
⭐ suck

Reply:
Agreed, but also know there were cases with Authenticator where customers wanted to translate similar strings on different screens in different ways. For now I was just trying to get the strings out of jsx and into one place where we can refactor them.